lib/routes/lnurl/payment/processing_lnurlp_dialog.dart

```diff
@@ -0,0 +1,145 @@
+import 'dart:async';
+
+import 'package:c_breez/widgets/payment_dialogs/processing_payment/processing_payment_animated_content.dart';
+import 'package:c_breez/widgets/payment_dialogs/processing_payment/processing_payment_content.dart';
+import 'package:flutter/material.dart';
+
+const _kPaymentListItemHeight = 72.0;
+
+class ProcessingLNURLPaymentDialog extends StatefulWidget {
```

Comment:
This seems like a duplication of the processing payment dialog we have right now. What is the advantage of this rather than use the existing one?

Reply:
I believe ProcessingPaymentDialog will get complicated and hard to debug if we try to inject this logic there.

Reply:
Merged here https://github.com/breez/c-breez/pull/504/commits/88ef255e3f98531133e208d41911ed1b38a16a8f